build.gradle

```diff
@@ -16,6 +16,7 @@ plugins {
     id "com.github.ben-manes.versions" version "0.30.0"
     id "com.osacky.fulladle"
     id "org.jmailen.kotlinter" version "3.0.2"
+    id "org.jetbrains.kotlin.jvm" version "$kotlin_version"
```

Comment:
can you tell me more about this change? why do we need this?

Reply:
Yeah, that was one of my experiments and it was left unintentionally 👍  Thanks